---
layout: spotlight
categories: spotlights

title: Tracy Johnson, Ph.D.
subtitle: Maria Rowena Ross Chair of Cell Biology and Biochemistry (University of California, Los Angeles)

headline: In 2014, Dr. Johnson was recognized for her leadership in both research and teaching as a Howard Hughes Medical Institute Professor. This honor includes a grant of $1 million over 5 years to develop programs that better integrate undergraduate education into research.

featured-image: /spotlights/images/tracy-johnson.jpeg
date: 2018-02-01
---

Dr. Johnson earned her Ph.D. in Biochemistry and Molecular Biology from UC Berkeley and did postdoctoral work at the California Institute of Technology, which is where she began to work on RNA splicing. She received the National Science Foundation’s CAREER award in 2005 and the Presidential Early Career Award for Scientists and Engineers in 2006. After her postdoc, she joined the faculty of UC San Diego, where she was awarded the Chancellor's Associates Award for Excellence in Undergraduate Teaching in 2013. That same year, she was also elected as one of the Top 20 Women Professors in California and moved her lab to UCLA. Here you can watch a <a class="light-bg" href="https://www.youtube.com/watch?v=cNu7qSag9f8" target="_blank" rel="noopener noreferrer">video</a> made about her Chancellor's Associates Award.

Within the genome, most genes are split up into multiple pieces: sequences that code for the protein (exons) are interrupted by non-coding sequences (introns). After the gene is transcribed into RNA, the introns must be removed so that the exons can be joined together to create the full coding sequence. This process, which creates a messenger RNA (mRNA), is called RNA splicing and is performed by the spliceosome. It is estimated that ~25% of mutations that cause genetic diseases are problematic because they change the ability of the spliceosome to recognize where the exon-intron boundaries are.

Historically, it was thought that the full RNA was transcribed first, and then splicing converted it into an mRNA. However, more recent evidence suggests that splicing and transcription are occurring at the same time. This co-transcriptional splicing opens up interesting questions about how the transcription machinery (RNA polymerase II) communicates and coordinates with the splicing machinery (the spliceosome).

Under a co-transcriptional splicing model, one challenge is to explain the spatial arrangement of everything. DNA is packaged in a highly-compact structure called chromatin, and while the pre-mRNA is being transcribed, it is closely enmeshed with the chromatin. Thus, we might predict that it is not accessible to the spliceosome to act on. This problem is one that Dr. Johnson and her lab are working to address, using yeast as a model. Budding yeast (<i>S. cerevisiae</i>) is a good system for studying RNA splicing because the majority of genes do not contain introns — this allows her to look specifically at proteins that interact with intron-containing genes more often than with other genes.

In a <a class="light-bg" href="http://doi.org/10.1101/gad.295188.116" target="_blank" rel="noopener noreferrer">recent paper</a>, Dr. Johnson focused on one of the proteins involved in packaging the DNA (histone variant H2A.Z). She showed that in yeast lacking this protein, the spliceosome is less good at changing conformation (shape), which it needs to do in order to bring the two exon edges close together and attach them. Specifically, this happened in places where splicing was predicted to be less ideal based on the RNA sequence. Previous studies had shown that this same protein is important for allowing transcription to move through condensed chromatin structures. Thus, Dr. Johnson concluded that this protein, which is highly conserved across the tree of life, plays an important role in coordinating RNA transcription with RNA splicing.